src/cljs/witan/ui/controllers/user.cljs

```diff
@@ -59,6 +65,8 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(declare login)
```

Comment:
`login` is used on line 93 but isn't defined until 112. `login` invokes `route-api-response -> api-response` and one of the `api-response` implementations calls `login`. Typical circular.